### DISCUSSION TITLE
Date picker require not working properly

### DISCUSSION THREAD

**Discussion by aldhahouri** [heroui-inc/heroui#2894] [General]
I've designed a form in my app to manage tasks, including a feature that allows users to specify a start time. This is facilitated through a date picker component. However, despite setting the 'IsRequired' attribute for the date picker, the form does not enforce this requirement, allowing users to proceed without selecting a date.

**Comment by HichTala** [comment]
I have the same problem. Have you managed to solve it?

**Reply by wingkwong** [reply]
Which version are you using? If it is latest, you need to handle the validation yourself since the default validation behaviour is aria. You may check out 2.4.0 blog breaking change section.
